dockerfiles/che/entrypoint.sh

```diff
@@ -324,14 +324,60 @@ add_che_cert_to_truststore() {
 }
 
 add_public_cert_to_truststore() {
+  JAVA_TRUST_STORE=/home/user/cacerts
+  DEFAULT_JAVA_TRUST_STOREPASS="changeit"
+
+  chmod 644 $JAVA_TRUST_STORE || true
```

Comment:
On openshift, need to make sure you have permission to write to this file before you attempt to change permissions or add content, eg., in Dockerfile before running entrypoint, must do:

```
    chmod 777 /home/user/cacerts
```

See https://github.com/eclipse/che/pull/17563/files#diff-8986c555679543224df189e631de16dfe70f3e624c4704c0c04c5f900a5ebd58R36

Reply:
Also when you're done making changes to the trust store, it's a good idea to make the file uneditable by the `user` user again, so that no more certs can be imported in that container. 

```
  chmod 444 $JAVA_TRUST_STORE || true

```
See https://github.com/eclipse/che/pull/17563/files#diff-0494d7f08e22baf0f7a6a136eb6f5a6a6532d3199d4edda56964dae41c18b22bR310-R311

Reply:
@nickboldt but how it works now on Openshift without my changes and `chmod 777`?

Reply:
because you're creating the file during entrypoint as anyuid user, I suspect we no longer need to worry about files created by root and updated by a user who's not in the root ownership/group memebership.